### ISSUE TITLE
Content not included in landmarks

### ISSUE THREAD

**Issue by JuliannaR** [wet-boew/GCWeb#1500]
**1.3.1 Info and Relationships**
HTML5 or WAI-ARIA landmarks are used on the page, but not all content is included.

When using HTML5 or WAI-ARIA landmarks it is best practice to include all content on the page in landmarks. In this way users of assistive technologies can use the landmarks for navigating the page without losing track of content.

Make sure that all content on the page is included in HTML5 or WAI-ARIA landmarks.

**How do I meet this criteria?**
ARIA11 Using ARIA landmarks to identify regions of a page

#accessibility #testing #adaptive #assistivetech
<img width="1434" alt="screen shot 2019-02-05 at 4 05 23 pm" src="https://user-images.githubusercontent.com/35537308/52304108-e01ade80-295f-11e9-9a8f-b628f17c6fb0.png">

@bsouster, @duboisp, @patlaj

**Comment by mgifford** [comment]
Yup https://dequeuniversity.com/rules/axe/3.1/region?application=AxeChrome

**Comment by shawnthompson** [comment]
This brings up the debate about conforming to WCAG standards and making web content 100% accessible. This is a WCAG best practice but not mandatory. I am **NOT** fighting for only do the minimal to pass WCAG. I face this every day while auditing web content and I have to try to show where it **fails** WCAG even though we can all see it's not accessible to everyone. GoC needs to come up with their own best practices and make enforce people to use them!

**Comment by mgifford** [comment]
Yes.. But frankly some pages should be held to a higher criterion than others I figure.  This is the home page of Canada.ca - let's make sure we're as good as possible.  If there's a way to make it easier for 20 more Canadians to use, let's find a way to make that happen.  It's the page that most folks will go to if they are looking for the gold standard for accessibility in the GC.

**Comment by patlaj** [comment]
Yes agreed... I'm not sure of the details but it seems like this falls in the "why would you not?" category.

**Comment by shawnthompson** [comment]
I think I found a fix for this "error" from aXe. If we put a `nav` element around the Skip to **navigation**. Stay tuned for an upcoming PR.

**Comment by shawnthompson** [comment]
Correction: I'm going to fix it in WET base. Stay tuned for that PR.

**Comment by shawnthompson** [comment]
Looks like I lied! I had to make the changes in both repos!
